ImperatorToCK3/Data_Files/configurables/nickname_map.txt

```diff
@@ -31,50 +31,50 @@ link = { imp = NICKNAME_CURIO imp = NICKNAME_THE_DEVOUT ck3 = nick_the_pious }
+link = { imp = NICKNAME_ITALICUS imp = NICKNAME_THE_ITALIOTE ck3 = nick_the_italian }
+link = { imp = NICKNAME_ILLYRICUS imp = NICKNAME_THE_ILLYRIAN ck3 = nick_the_dalmatian }
 link = { imp = NICKNAME_HISPANICUS imp = NICKNAME_THE_IBERIAN ck3 = nick_the_motherfather_of_spain }
-#link = { imp = NICKNAME_MAURETANICUS imp = NICKNAME_THE_MAURETANIAN ck3 = ??? } #'The Mauretanian'
-#link = { imp = NICKNAME_NUMIDIUS imp = NICKNAME_THE_NUMIDIAN ck3 = ??? } #'The Numidian'
+link = { imp = NICKNAME_MAURETANICUS imp = NICKNAME_THE_MAURETANIAN ck3 = nick_the_moor }
+link = { imp = NICKNAME_NUMIDIUS imp = NICKNAME_THE_NUMIDIAN ck3 = nick_the_algerian }
 link = { imp = NICKNAME_VENEDICUS imp = NICKNAME_THE_VENEDIAN ck3 = nick_the_wend }
-#link = { imp = NICKNAME_DACIUS imp = NICKNAME_THE_DACIAN ck3 = ??? } #'The Dacian'
-#link = { imp = NICKNAME_HELLENICUS imp = NICKNAME_THE_HELLENE ck3 = ??? } #'The Hellene'
-#link = { imp = NICKNAME_CYRENAICUS imp = NICKNAME_THE_CYRENE ck3 = ??? } #'The Cyrene'
-#link = { imp = NICKNAME_AEGYPTUS imp = NICKNAME_THE_EGYPTIAN ck3 = ??? } #'The Egyptian'
+link = { imp = NICKNAME_DACIUS imp = NICKNAME_THE_DACIAN ck3 = nick_the_romanian }
+link = { imp = NICKNAME_HELLENICUS imp = NICKNAME_THE_HELLENE ck3 = nick_the_greek }
+link = { imp = NICKNAME_CYRENAICUS imp = NICKNAME_THE_CYRENE ck3 = nick_the_libyan }
+link = { imp = NICKNAME_AEGYPTUS imp = NICKNAME_THE_EGYPTIAN ck3 = nick_the_egyptian }
 link = { imp = NICKNAME_NUBIANUS imp = NICKNAME_THE_NUBIAN ck3 = nick_the_black }
-#link = { imp = NICKNAME_ARABICUS imp = NICKNAME_THE_ARABIAN ck3 = ??? } #'The Arabian'
-#link = { imp = NICKNAME_SYRIANUS imp = NICKNAME_THE_SYRIAN ck3 = ??? } #'The Syrian'
-#link = { imp = NICKNAME_ASIATICUS imp = NICKNAME_THE_ASIAN ck3 = ??? } #'The Asian'
-#link = { imp = NICKNAME_SARMATICUS imp = NICKNAME_THE_SARMATIAN ck3 = ??? } #'The Sarmatian'
-#link = { imp = NICKNAME_BOSPORANUS imp = NICKNAME_THE_BOSPORAN ck3 = ??? } #'The Bosporan'
-#link = { imp = NICKNAME_CAUCASIANUS imp = NICKNAME_THE_CAUCASIAN ck3 = ??? } #'The Caucasian'
-#link = { imp = NICKNAME_SCYTHIANUS imp = NICKNAME_THE_SCYTHIAN ck3 = ??? } #'The Scythian'
-#link = { imp = NICKNAME_PERSICUS imp = NICKNAME_THE_PERSIAN ck3 = ??? } #'The Persian'
-#link = { imp = NICKNAME_BACTRIANUS imp = NICKNAME_THE_BACTRIAN ck3 = ??? } #'The Bactrian'
-#link = { imp = NICKNAME_GEDROSIANUS imp = NICKNAME_THE_GEDROSIAN ck3 = ??? } #'The Gedrosian'
-#link = { imp = NICKNAME_INDICUS imp = NICKNAME_THE_INDIAN ck3 = ??? } #'The Indian'
-#link = { imp = NICKNAME_DRAVIDICUS imp = NICKNAME_THE_DRAVIDIAN ck3 = ??? } #'The Dravidian'
+link = { imp = NICKNAME_ARABICUS imp = NICKNAME_THE_ARABIAN ck3 = nick_the_arabian }
+link = { imp = NICKNAME_SYRIANUS imp = NICKNAME_THE_SYRIAN ck3 = nick_the_syrian }
+link = { imp = NICKNAME_ASIATICUS imp = NICKNAME_THE_ASIAN ck3 = nick_the_asian }
+link = { imp = NICKNAME_SARMATICUS imp = NICKNAME_THE_SARMATIAN ck3 = nick_the_russian }
```

Comment:
Sarmatians were Iranian peoples, so does it make sense to convert to "the Russian"?

Reply:
Ditto